### ISSUE TITLE
MathBlock should be $$% to be compliant with GH crap

### ISSUE THREAD

**Issue by Ducasse** [pillar-markup/Microdown#457]
$$%caption=This is a super cool formula&anchor=eq1
\left( \sum_{k=1}^n a_k b_k \right)^2 \leq \left( \sum_{k=1}^n a_k^2 \right) \left( \sum_{k=1}^n b_k^2 \right) 
$$

**Comment by Ducasse** [comment]
```
$$%caption=This is a super cool formula&anchor=eq1
\left( \sum_{k=1}^n a_k b_k \right)^2 \leq \left( \sum_{k=1}^n a_k^2 \right) \left( \sum_{k=1}^n b_k^2 \right) 
$$
```

**Comment by Ducasse** [comment]
The above is an example of the way we should write caption and anchor in microdown.

**Comment by kasperosterbye** [comment]
Fixed by PR #455 and #456
